.github/workflows/stop-goth.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo STOP GOTH
+kill $(ps -aux | grep "[g]oth start" | awk '{print $2}')
+
+echo STOP DOCKER CONTAINERS
+docker rm --force $(docker ps -a -q)
```

Comment:
I wonder if we can get PID from this
```
python -m goth start .envs/goth/assets/goth-config.yml &
GOTH_PID=$!
```

Or if there is a symetric command to the `goth start`